static/app/views/replays/replayTable/index.tsx

```diff
@@ -42,14 +45,36 @@ function ReplayTable({
   sort,
   visibleColumns,
   emptyMessage,
+  saveLocation,
+  headersSortable,
 }: Props) {
   const routes = useRoutes();
-  const location = useLocation();
+  const newLocation = useLocation();
   const organization = useOrganization();
 
+  const location: Location =
+    saveLocation ?? false
```

Comment:
nit: I'm not sure you need the `?? false` part

Research:
`??` is the [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing), it only cares about `null` and `undefined`.
Whereas the ternary `a ? b : c` checks for [truthy-ness](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) of `a`, which includes null and undefined, and a bunch more.